### PR TITLE
fixing the IsExpired function. Not sure what I was thinking but it ha…

### PR DESCRIPTION
**Pull request by endophage**
…d something to do with the PyPI testing

@dmcgowan @NathanMcCauley

### CODE REVIEW

File: signed/verify_test.go [hunk 1]
```diff
@@ -132,7 +132,7 @@ func Test(t *testing.T) {
 			role: "root",
 			name: "expired",
 			exp:  &expiredTime,
-			err:  ErrExpired{"root", expiredTime.Format("2006-01-02 15:04:05 MST")},
+			err:  ErrExpired{"root", expiredTime.Format("Mon Jan 2 15:04:05 MST 2006")},
```
Comment (dmcgowan) [inline signed/verify_test.go:135]: Which standard is this supposed to be using and can we just use a constant defined in the `time` package. Looks almost identical to `time.UnixDate`

File: signed/verify.go [hunk 2]
```diff
@@ -85,7 +85,7 @@ func verifyMeta(s *data.Signed, role string, minVersion int) error {
 	}
 	if IsExpired(sm.Expires) {
 		logrus.Errorf("Metadata for %s expired", role)
-		return ErrExpired{Role: role, Expired: sm.Expires}
+		return ErrExpired{Role: role, Expired: sm.Expires.Format("Mon Jan 2 15:04:05 MST 2006")}
```
Comment (dmcgowan) [inline signed/verify.go:88]: Same comment as below

File: signed/verify_test.go (hunk 1 above)
Comment (endophage) [inline signed/verify_test.go:135]: This is just meant to be human readable (it gets output somewhere I think, I know I've seen it in the terminal). The default format includes milliseconds.

File: signed/verify.go (hunk 2 above)
Comment (endophage) [inline signed/verify.go:88]: Yeah, just meant to be human readable here.

**Comment by dmcgowan** [comment]
LGTM

**Pull request merged**